src/baseline.rs

```diff
@@ -7,22 +8,196 @@ use crate::dump::RustDocCommand;
 use crate::util::slugify;
 use crate::GlobalConfig;
 
+fn get_all_crate_features_from_registry(crate_: &crates_index::Version) -> Vec<String> {
+    // Sometimes crates ship types with fields or variants that are included
+    // only when certain features are enabled.
+    //
+    // By default, we want to generate rustdoc with `--all-features`,
+    // but that option isn't available here so we have to implement it ourselves.
+    // TODO: this comment is not yet true, but it'll be in some next PR. Then move the whole comment
+    // outside and remove this TODO.
```

Comment:
What's not true about this? The code seems to me to do what it says at the moment?

Reply:
Oh, indeed, it is true. So the only thing left is to somehow lift the comment up so it references both the `get_all_crate_features_*` functions.

Reply:
Move the two functions into their own submodule and make this kind of thing a comment on the module itself?

Reply:
Done in f54a285b77811935d4ecdee3ab128b1575d980f5.